main.nf

```diff
@@ -1,299 +1,80 @@
-n${params.n_secondary}-\
-a${params.align_pct_id}-\
-K${params.mash_kmer}-\
-k${params.min_match_length}-\
-w${params.max_block_weight}-\
-j${params.max_path_jump}-\
-W${params.min_subpath}-\
-e${params.max_edge_jump}\
-""" }
-
-fasta = Channel.fromPath("${params.input_fasta}").map { f -> tuple(makeBaseName(f), f) }
-
-process edyeet {
-  input:
-  tuple val(f), file(fasta)
-
-  output:
-  tuple val(f), file(fasta), file("${f}.paf")
-
-  """
-  edyeet -X \
-     -s ${params.segment_length} \
-     -p ${params.map_pct_id} \
-     -n ${params.n_secondary} \
-     -a ${params.align_pct_id} \
-     -k ${params.mash_kmer} \
-     -t ${task.cpus} \
-     $fasta $fasta \
-     >${f}.paf 
-  """
-}
-
-
-process seqwish {
-  input:
-  tuple val(f), file(fasta), file(alignment)
-
-  output:
-  tuple val(f), file("${f}.seqwish.gfa")
-
-  """
-  seqwish \
-    -t ${task.cpus} \
-    -s $fasta \
-    -p $alignment \
-    -k ${params.min_match_length} \
-    -g ${f}.seqwish.gfa -P
-  """
-}
+// We dont't need this parameter in this pipeline
+// params.fasta = WorkflowMain.getGenomeAttribute(params, 'fasta')
 
-process smoothxg {
-  input:
-  tuple val(f), file(graph)
-
-  output:
-  tuple val(f), file("${f}.smooth.gfa")
-
-  """
-  smoothxg \
-    -t ${task.cpus} \
-    -g $graph \
-    -w ${params.max_block_weight} \
-    -j ${params.max_path_jump} \
-    -e ${params.max_edge_jump} \
-    -l ${params.max_poa_length} \
-    -o ${f}.smooth.gfa \
-    -m ${f}.smooth.maf \
-    -s ${f}.consensus \
-    -a \
-    -C 5000 \
-  """
-}
-
-process odgiBuild {
-  input:
-  tuple val(f), file(graph)
-
-  output:
-  tuple val(f), file("${f}.smooth.og")
-
-  """
-  odgi build -g $graph -o ${f}.smooth.og
-  """
-}
-
-process odgiViz {
-  input:
-  tuple val(f), file(graph)
-
-  output:
-  tuple val(f), file("${f}.smooth.og.viz.png")
-
-  """
-  odgi viz \
-    -i $graph \
-    -o ${f}.smooth.og.viz.png \
-    -x 1500 -y 500 -P 5
-  """
-}
-
-process odgiChop {
-  input:
-  tuple val(f), file(graph)
-
-  output:
-  tuple val(f), file("${f}.smooth.chop.og")
-
-  """
-  odgi chop -i $graph -c 100 -o ${f}.smooth.chop.og
-  """
-}
-
-process odgiLayout {
-  input:
-  tuple val(f), file(graph)
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    VALIDATE & PRINT PARAMETER SUMMARY
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
 
-  output:
-  tuple val(f), file(graph), file("${f}.smooth.chop.og.lay")
+include { validateParameters; paramsHelp } from 'plugin/nf-validation'
 
-  """
-  odgi layout \
-    -i $graph \
-    -o ${f}.smooth.chop.og.lay \
-    -t ${task.cpus} -P
-  """
+// Print help message if needed
+if (params.help) {
+    def logo = NfcoreTemplate.logo(workflow, params.monochrome_logs)
+    def citation = '\n' + WorkflowMain.citation(workflow) + '\n'
+    def String command = "nextflow run ${workflow.manifest.name} --input samplesheet.csv --genome GRCh37 -profile docker"
```

Comment:
This example command string does not work for pangenome

Reply:
Perhaps something like in the readme
```bash
nextflow run ${workflow.manifest.name} --input <BGZIPPED_FASTA> --n_haplotypes <NUM_HAPS_IN_FASTA> --outdir <OUTDIR> -profile docker
```

Reply:
For sure this was introduced with the new Template update.